pkg/cmdline/env_test.go

```diff
@@ -70,11 +70,11 @@ func TestEnvStringNSlice(t *testing.T) {
 	v = cmd.Flag("stringSlice").Value.String()
 	assert.Equal(t, v, "[]", "The flag should be unset.")
 
-	envStringNSlice(cmd.Flag("stringSlice"), "sliceval")
+	EnvStringNSlice(cmd.Flag("stringSlice"), "sliceval")
 	v = cmd.Flag("stringSlice").Value.String()
 	assert.Equal(t, v, "[sliceval]", "The flag should be set to value provided.")
 
-	envStringNSlice(cmd.Flag("stringSlice"), "newsliceval")
+	EnvStringNSlice(cmd.Flag("stringSlice"), "newsliceval")
```

Comment:
interesting... there should be tests that test the slice part of "EnvStringSlice", all these tests are looking at a single string.

Reply:
Will rewrite the whole tests for env